cs/test/expressions/schemas.bond

```diff
@@ -12,6 +12,8 @@ struct Nested
     0: double _double;
 }
 
+using decimal = int32;
```

Comment:
A decimal can't always fit in an int32. I would have expected a blob here... Am I missing something?

Reply:
Nope, just a mistake on my part. This was something I was playing with to compare behavior when aliasing a primitive vs a blob and I didn't mean to include it in the PR. Reverted.